src/utils/CameraUtils.js

```diff
@@ -0,0 +1,448 @@
+import * as THREE from 'three';
+import TWEEN from '@tweenjs/tween.js';
+import DEMUtils from './DEMUtils';
+import { MAIN_LOOP_EVENTS } from '../Core/MainLoop';
+import Coordinates from '../Core/Geographic/Coordinates';
+
+THREE.Object3D.DefaultUp.set(0, 0, 1);
+const targetPosition = new THREE.Vector3();
+const targetCoord = new Coordinates('EPSG:4326', 0, 0, 0);
+const rigs = [];
+const slerp = [];
+
+const deferred = () => {
+    let resolve;
+    return { promise: new Promise((r) => { resolve = r; }), resolve };
+};
+
+// Wrap angle in degrees to [-180 180]
+function wrapTo180(angle) {
+    return angle - Math.floor((angle + 179.0) / 360) * 360;
```

Comment:
What about `(angle % 360) - 180` ?

Reply:
if angle is -181, it doesn't work
we are getting
 * `(angle % 360) - 180 // => -361`
 * `angle - Math.floor((angle + 179.0) / 360) * 360 // => 179

Reply:
`angle - Math.floor((angle + 179.0) / 360) * 360` is not good either (as expected with a magic number 179 degrees !):  angle =180.5 returns 180.5 instead of -179.5
A correct formula would be `angle - Math.floor((angle + 180.0) / 360) * 360`

Reply:
thanks